content/blog/restic-0.17.0-released.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Restic 0.17.0 Released"
+date: 2024-07-25T00:00:00+02:00
+---
+
+We are happy to announce the release of [restic 0.17.0](https://github.com/restic/restic/releases/v0.17.0)!
+
+After almost a year of hard work, this release brings a lot of improvements, of which the main highlights are:
+
+- Highly improved [`restore`](https://restic.readthedocs.io/en/stable/050_restore.html) command, now supporting [dry-run](https://restic.readthedocs.io/en/stable/050_restore.html#dry-run), [in-place restore](https://restic.readthedocs.io/en/stable/050_restore.html#restoring-in-place) with different overwrite modes and a [delete](https://restic.readthedocs.io/en/stable/050_restore.html#delete-files-not-in-snapshot) option.
+- Resumable and optimized [`prune`](https://restic.readthedocs.io/en/stable/060_forget.html) command, now using up to 60% less memory and being less write-intensive.
+- Ability to [skip snapshot creation if nothing changed](https://restic.readthedocs.io/en/stable/040_backup.html#skip-creating-snapshots-if-unchanged).
+- Support for [repositories with empty password](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#repositories-with-empty-password).
+- Support for [FUSE-T on macOS](https://restic.readthedocs.io/en/stable/050_restore.html#restore-using-mount).
+- Support for [size](https://restic.readthedocs.io/en/latest/045_working_with_repos.html#listing-all-snapshots) and [summary](https://restic.readthedocs.io/en/latest/075_scripting.html#summary) statistics in snapshots.
```

Comment:
These links should also use `stable` instead of `latest`.

Reply:
Fixed, thanks for noticing!